### PR TITLE
Fix theme issues

### LINKED ISSUE #410: Theme Issues
A collection of issues regarding the dark/light theme, also see [icingaweb2/#4565](https://github.com/Icinga/icingaweb2/issues/4565)

- [x] Controls in light theme
- [x] List item text color in light theme
- [x] Plugin output background-color
- [x] object-meta-info-control background in light theme
- [x] Check Statistics in light theme
- [x] Group tags in light theme
- [x] Detail Header:  list-item text color
- [x] Detail Header: state-ball background should be set to `@body-bg-color`
- [x] Handled problem state-ball icons should be set to `@body-bg-color`
- [x] Handled state-ball background-color should avoid opacity
- [x] List footer color in light theme
- [x] Feature command `label`s in light theme
- [x] Downtime progress widget in light theme
- [x] Downtime Detail Delete Button
- [x] Health View
- [x] Vars default levels don’t seem to be applied correctly, when in light theme
- [x] Overdue list items background in light theme
- [x] Migrate Popup background in light theme
- [x] `.control-button` hover background
- [x] Host/Service Detail label/value contrast

### Controls in light theme
Screen Shot 2021-10-20 at 14 35 46

### List item text color in light theme
Screen Shot 2021-10-20 at 14 36 51

### Plugin output background-color
Screen Shot 2021-10-20 at 14 37 37

### object-meta-info-control background in light theme
Screen Shot 2021-10-20 at 14 39 32

### Check Statistics in light theme
Screen Shot 2021-10-20 at 14 39 51

### Group tags in light theme
Screen Shot 2021-10-20 at 14 40 18

### Detail Header
* list-item text color
* state-ball background should be set to `@body-bg-color`
Screen Shot 2021-10-20 at 14 40 55

### Handled problem state-ball icons should be set to `@body-bg-color`
Screen Shot 2021-10-20 at 14 46 32

#### Handled state-ball background-color should avoid opacity
It’s not relevant for lists, but in the `.state-change` element with overlapping `state-ball` elements it’s a problem.
Screen Shot 2021-10-20 at 14 46 59

### List footer color in light them
Screen Shot 2021-10-20 at 14 49 55

### Feature command `label`s in light theme
Screen Shot 2021-10-20 at 14 50 14

### Downtime progress widget in light theme
Screen Shot 2021-10-20 at 14 52 09

### Downtime Detail Delete Button
Screen Shot 2021-10-20 at 14 52 46

### Health View
* `.vertical-key-value .value` should be set to `@text-color`
Screen Shot 2021-10-20 at 14 54 11

### Vars default levels don’t seem to be applied correctly, when in light theme
Not sure if this is intentional?
See, the default value should be `#F5F9FA`(the light body-bg-color)
Screen Shot 2021-10-20 at 16 46 43

### Overdue list items background in light theme
Screen Shot 2021-10-21 at 13 13 25

### Migrate Popup background in light theme
Screen Shot 2021-10-21 at 13 22 44

### `.control-button` hover background
Screen Shot 2021-10-27 at 16 25 31

### Host/Service Detail label/value contrast
Screen Shot 2021-10-27 at 16 51 43

### PR DESCRIPTION
**Pull request by flourish86**
fixes #410

### CODE REVIEW

File: public/css/widget/state-change.less [hunk 1]
```diff
@@ -39,4 +39,69 @@
...
+      left: -2px;
+      right: -2px;
+      bottom: -2px;
+      border-radius: 50%;
+      .var(background-color, body-bg-color);
+      z-index: 1;
+    }
+
+    &.state-pending:before {
+      background-color: @color-pending;
```
Comment (nilmerg) [inline public/css/widget/state-change.less:80]: This and the other background colors are not using the css `var` function. (You don't have to use the `.vars` mixin, just `var(--color-pending, @color-pending)` is sufficient.)

File: public/css/widget/state-change.less [hunk 2]
```diff
@@ -39,4 +39,69 @@
...
+    }
+
+    &.state-unknown:before {
+      background-color: @color-warning;
+    }
+  }
+}
+
+.overdue .state-change .state-ball {
+  .box-shadow(0, 0, 0, 1px, ~"var(--gray-lighter, @{gray-lighter}) !important");
```
Comment (nilmerg) [inline public/css/widget/state-change.less:102]: Why important? Is fine without it for me at least.

**Review by nilmerg — CHANGES_REQUESTED** [review]

File: public/css/widget/state-change.less (hunk 1 above)
Comment (flourish86) [reply public/css/widget/state-change.less:80]: ✅

File: public/css/widget/state-change.less (hunk 2 above)
Comment (flourish86) [reply public/css/widget/state-change.less:102]: ✅

**Review by nilmerg — APPROVED** [review]

**Pull request merged**